src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -3,16 +3,22 @@ import React, { useEffect } from 'react';
 import styles from './HeatmapVis.module.css';
 import ColorBar from './ColorBar';
 import { useHeatmapStyles, useProps, useValues } from './hooks';
-import AxisGrid from './AxisGrid';
+import AxisGrid from '../shared/AxisGrid';
 import Mesh from './Mesh';
-import Tooltip from './Tooltip';
+import Tooltip from '../shared/Tooltip';
 import HeatmapProvider from './HeatmapProvider';
 import { useHeatmapConfig } from './config';
+import { Domain } from '../shared/models';
+import PanZoomMesh from '../shared/PanZoomMesh';
```

Comment:
Should we rename `Tooltip` to `TooltipMesh` as well, then? Either that or we just name this one `PanZoom`.

Reply:
`TooltipMesh` that is